x-pack/plugins/enterprise_search/server/routes/app_search/log_settings.test.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { MockRouter, mockRequestHandler, mockDependencies } from '../../__mocks__';
+
+import { registerLogSettingsRoutes } from './log_settings';
+
+describe('log settings routes', () => {
+  describe('PUT /api/app_search/log_settings', () => {
```

Comment:
I don't have super strong feelings and this is not a blocker, but I might suggest generally putting ordering our GET endpoint to follow both user flow and the source code, and the PUT tests after

Reply:
Good suggestion, I'll do that

Reply:
97193e8a743b53cd07fc544e0e5da0d243c69c20